.gitignore

```diff
@@ -11,5 +11,8 @@
 /GoTrue/build/
 /Functions/build/
 /plugins/ApolloGraphQL/build
+/plugins/ComposeAuthUI/build/
+/plugins/ComposeAuth/build/
+/serializers/
```

Comment:
Now the whole serializers directory gets ignored, probably have to write out the paths too

Reply:
> Now the whole serializers directory gets ignored, probably have to write out the paths too

Just updated. What do you mean by mentioning _write out the paths_ ?

Reply:
No, I just mean the serializers directory (/serializiers/Jackson/build/ etc.)

Reply:
> No, I just mean the serializers directory (/serializiers/Jackson/build/ etc.)

Updated. Just wonder why do we need to specify for each serializer? If we introduce more, we need to update ignore again?

Reply:
Probably, the serializers also generate build files

Reply:
Make sense